fabulaws/library/wsgiautoscale/servers.py

```diff
@@ -233,15 +233,15 @@ class DbMixin(PostgresMixin):
     """Mixin that creates a database based on the Fabric env."""
 
     # use the default version in this Ubuntu distro
-    postgresql_packages = ['postgresql', 'libpq-dev']
-    postgresql_tune = True
-    postgresql_shmmax = 107374182400 # 100 GB
-    postgresql_shmall = 26214400 # 100 GB / PAGE_SIZE (4096)
+    postgresql_packages = env.get('postgresql_packages', ['postgresql', 'libpq-dev'])
```

Comment:
Could we pass these in through ``kwargs`` rather than tapping into env directly, similar to ``BaseInstance.__init__``?